00-HTML-CSS-basics/CSS/css/grid.css

```diff
@@ -0,0 +1,193 @@
+/* Small screens (default) */
+html {
+    font-size: 100%;
+    font-family: "Calibri", serif;
+}
+
+/* Medium screens (640px) */
+@media (min-width: 40rem) {
+    html { font-size: 112%; }
```

Comment:
Is a possible use case to use percentages into font-sizes. Anyway I prefer pixels :P